dio/pubspec.yaml

```diff
@@ -1,9 +1,10 @@
 name: diox
-description: A powerful Http client for Dart, which supports Interceptors, FormData, Request Cancellation, File Downloading, Timeout etc.
-version: 5.0.0-dev.3
+
+description: A powerful HTTP Client for Dart/Flutter, which supports Global Settings, Interceptors, FormData, Aborting a Request, File Uploading/Downloading, Timeout and Custom Adapters, etc.
 homepage: https://github.com/cfug/diox
 repository: https://github.com/cfug/diox/blob/main/dio
 issue_tracker: https://github.com/cfug/diox/issues
+version: 5.0.0-dev.3
```

Comment:
What's the point of moving this field?

Reply:
Just checked out serval official plugins maintained by Flutter team, like [url_launcher](https://github.com/flutter/plugins/blob/main/packages/url_launcher/url_launcher/pubspec.yaml), [image_picker](https://github.com/firebase/flutterfire/blob/master/packages/cloud_firestore/cloud_firestore/pubspec.yaml), [flutterfire](https://github.com/firebase/flutterfire/blob/master/packages/cloud_firestore/cloud_firestore/pubspec.yaml), found they put the version code under the `issue_tracker` 😂 Feel free to change it back.